commands/channel_e2e_test.go

```diff
@@ -54,3 +57,117 @@ func (s *MmctlE2ETestSuite) TestUnarchiveChannelsCmdF() {
 		s.Require().Contains(printer.GetErrorLines()[0], "Unable to unarchive channel. The channel is not archived.")
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestMoveChannelCmd() {
+	s.SetupTestHelper().InitBasic()
+	initChannelName := api4.GenerateTestChannelName()
+	var appErr *model.AppError
+	var channel *model.Channel
+	var team *model.Team
```

Comment:
To improve readability, could we use here the `:=` assignment with these variables instead of declaring them first?

Reply:
ya removed that i added it  because i got some govet error- shadow: declaration of "appErr" shadows declaration at line 64 (govet), i made some changes and pushed for this issue also u can check